pkg2/lib/bar.ml

```diff
@@ -1,3 +1,3 @@
-open Foo
+open Pkg1.Foo
```

Comment:
Dune wraps libraries in the `public_name`. you can use `(wrapped false)` if you want the toplevel namespace to be `Foo`. Or alternatively name the library `foo`